hack/start-agent.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# arguments: TARGET, YGGDRASIL_PATH, (optional) DEVICE_WORKER_PATH
+for ARGUMENT in "$@"
+do
+   KEY=$(echo $ARGUMENT | cut -f1 -d=)
+   KEY_LENGTH=${#KEY}
+   VALUE="${ARGUMENT:$KEY_LENGTH+1}"
+   export "$KEY"="$VALUE"
+done
+
+if [[ -z $TARGET ]]; then
+    echo "Error: TARGET is required"
+    exit 1
+fi
+
+if [[ -z $YGGDRASIL_PATH ]]; then
+    echo "Error: YGGDRASIL_PATH is required"
+    exit 1
+fi
+
+if [[ $# = 3 ]]; then
+  cd $DEVICE_WORKER_PATH
+fi
+make uninstall
+make install
+
+cd $YGGDRASIL_PATH
+git checkout cdde836c519ac72302714b5f7ae90e45de7c79a7
+export PREFIX=/var/local
+make bin
+sudo chown root.root /tmp/*.pem
+
+pid_file = "/var/local/var/run/yggdrasil/workers/device-worker.pid"
```

Comment:
pls use PREFIX as prefix for this:
`pid_file="${PREFIX}/var/run/yggdrasil/workers/device-worker.pid"`
also, no spaces before and after the assignment operator

Reply:
done